lists-service/helm/templates/config.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "ala-species-lists.fullname" . }}
+  labels:
+    {{- include "ala-species-lists.labels" . | nindent 4 }}
+data:
+{{ (tpl (.Files.Glob "config/lists-service-config.properties").AsConfig . ) | indent 2 }}
+{{ (.Files.Glob "config/constraints.json").AsConfig | indent 2 }}
+binaryData:
+  lists-service-truststore.jks: |-
+{{ .Files.Get "config/lists-service-truststore.jks" | b64enc | indent 4 }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "ala-species-lists.fullname" . }}
+  labels:
+    {{- include "ala-species-lists.labels" . | nindent 4 }}
+data:
+  MONGODB_USERNAME: {{ .Values.mongodb.username | b64enc }}
+  MONGODB_PASSWORD: {{ .Values.mongodb.password | b64enc }}
+  OIDC_CLIENT_ID: {{ .Values.oidc.clientId | b64enc }}
```

Comment:
all these values are in config.ini and are made available to the pod outside of Helm